frontend/src/views/Transfers/components/TransferHistory.jsx

```diff
@@ -66,6 +66,11 @@ function TransferHistory({ transferHistory }) {
     category = 'C'
   }
 
+  // Filter out any DRAFT records, so “Created draft” never shows up
```

Comment:
Let's do this on the backend instead so the frontend just renders what it receives.